include/rangers-api/Application/Save/SaveManagerListener.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+namespace app::save {
+    class SaveManagerListener {
```

Comment:
Starting braces should be started on a line to keep consistent with the original repository.